docs/source/customizations/coprocessor.mdx

```diff
@@ -504,6 +504,7 @@ Properties of the JSON body are divided into two high-level categories:
   "stage": "SubgraphRequest",
   "control": "continue",
   "id": "666d677225c1bc6d7c54a52b409dbd4e",
+  "subgraphRequestId": "b5964998b2394b64a864ef802fb5a4b3",
```

Comment:
Could we show an example of yaml configuration to include that key ?

Reply:
[a07cafb](https://github.com/apollographql/router/pull/5858/commits/a07cafb7b1aa916236be15ed962b5e0873674e8f)